app/packages/state/src/routing/RouterContext.ts

```diff
@@ -263,6 +263,7 @@ async function fetchGraphQL(
   );
 
   if ("errors" in data && data.errors) {
+    console.error(data);
```

Comment:
Added intentionally as the error I was seeing was not actually displayed in the logs.